### PR TITLE
cleanUp

### PR DESCRIPTION
**Pull request by DNaunin**
Refactor Code, deleted unneeded files and updated ReadMe

### CODE REVIEW

**Review by JuKahl — APPROVED** [review]

**Review by RonjaMeinecke — APPROVED** [review]
🧼 LGTM

**Comment by JuKahl** [comment]
LGTM 👩‍🎨

File: README.md [hunk 1]
```diff
@@ -1,34 +1,23 @@
...
-
-- [Next.js Documentation](https://nextjs.org/docs) - learn about Next.js features and API.
-- [Learn Next.js](https://nextjs.org/learn) - an interactive Next.js tutorial.
-
-You can check out [the Next.js GitHub repository](https://github.com/vercel/next.js/) - your feedback and contributions are welcome!
-
-## Deploy on Vercel
-
-The easiest way to deploy your Next.js app is to use the [Vercel Platform](https://vercel.com/new?utm_medium=default-template&filter=next.js&utm_source=create-next-app&utm_campaign=create-next-app-readme) from the creators of Next.js.
+## Development
```
Comment (mewiemers) [inline README.md:18]: you can write your deployment on Heroku here, if you want

**Review by mewiemers — APPROVED** [review]
GJ!!😎

**Pull request merged**